### ISSUE TITLE
Testing of file existence is not working

### ISSUE THREAD

**Issue by fbergin** [stevelacy/gulp-git#132]
The code used to test for created files might not be ok. [Most](https://github.com/stevelacy/gulp-git/search?utf8=%E2%9C%93&q=%22should.exist%22&type=Code) of it is of the form: `should.exist('path/to/file');`. The [docs](http://shouldjs.github.io/#should-exist) for `should.exist(obj[, msg])` state:

> Assert obj exists, with optional message.

This means you can pass any string to the function and no exception will be thrown:

```javascript
> should.exist('');
undefined
> should.exist('file/which/does/not/exist');
undefined
```

**Comment by stevelacy** [comment]
Nice catch, PR?

**Comment by fbergin** [comment]
Sure, I'll see what I can do in the next few days.

**Comment by stevelacy** [comment]
Thanks, published to 1.11.1
